app/uk/gov/hmrc/tradergoodsprofilesrouter/models/response/GoodsItemRecords.scala

```diff
@@ -155,4 +155,21 @@ object GoodsItemRecords {
       case AccreditationStatus.Rejected => AdviceStatus.AdviceNotProvided
       case _                            => AdviceStatus.withName(accreditationStatus.entryName)
     }
+
+  private def translateReviewReason(reviewReason: Option[String], toReview: Boolean): Option[String] =
+    if (!toReview) None
```

Comment:
can you do this other way round like if (toReview) else None please? it's best we check positive condition first

Reply:
It is because we are only doing checks for the toReview before doing the match. we can have conversation about it